tests/ImageSharp.Tests/PixelFormats/RgbaVectorTests.cs

```diff
@@ -189,5 +189,21 @@ public void RgbaVector_FromGrey8()
             // assert
             Assert.Equal(expected, rgba.ToScaledVector4());
         }
+
+        [Fact]
+        public void Issue2048()
+        {
+            // https://github.com/SixLabors/ImageSharp/issues/2048
+            RgbaVector green = Color.Green.ToPixel<RgbaVector>();
```

Comment:
This is actually covered in the `PixelOperation` tests but I like to be explicit for serious bugs.